packages/fx-core/src/plugins/resource/sql/constants.ts

```diff
@@ -57,22 +57,27 @@ export class Constants {
 }
 
 export class Telemetry {
-    static readonly telemetryName = "fx-resource-azure-sql";
-    static readonly provisionStart = `${Telemetry.telemetryName}/provision-start`;
-    static readonly provisionEnd = `${Telemetry.telemetryName}/provision`;
-    static readonly postProvisionStart = `${Telemetry.telemetryName}/post-provision-start`;
-    static readonly postProvisionEnd = `${Telemetry.telemetryName}/post-provision`;
+    static readonly componentName = "fx-resource-azure-sql";
+    static startSuffix = "-start";
+    static resultYes = "yes";
+    static resultNo = "no";
+    static userError = "user";
+    static systemError = "system";
 
-    static readonly component = "component";
-    static readonly success = "success";
-    static readonly errorType = "error-type";
-    static readonly errorMessage = "error-message";
+    static readonly stage = {
+        preProvision: "pre-provision",
+        provision: "provision",
+        postProvision: "post-provision",
```

Comment:
do we have post-provision? Seems no such event in Changyong's doc

Reply:
pre and post event are included in the events.
![image](https://user-images.githubusercontent.com/16380704/117926231-06900700-b32b-11eb-8ff4-dedaa7c14521.png)